openmetadata-ui/src/main/resources/ui/src/axiosAPIs/miscAPI.ts

```diff
@@ -222,23 +221,20 @@ export const deleteEntity = async (
   entityType: string,
   entityId: string,
   isRecursive: boolean,
-  isSoftDelete = false
+  isHardDelete = true
```

Comment:
if this method you are using anywhere please test it out properly as we are making change in params

Reply:
Changes made accordingly and tested out in other places 👍